build_compiler.js

```diff
@@ -48,6 +50,7 @@ ncp('./compiler/contrib', './contrib', err => {
 });
 
 if (shouldDownloadCompiler) {
+  spawnSync('mvn', ['clean']);
```

Comment:
Is this not redundant with the gwtBuild below, or does clean in the context of pom-gwt use different output directories?

Reply:
The gwt build uses different output directories based off of the version number. I added this so that it wasn't possible to end up with more than one folder:

 - closure-compiler-gwt-1.0-SNAPSHOT
 - closure-compiler-gwt-v20180610

Reply:
Really this is defense for local development. Travis would never have more than one since it starts with a clean environment each time.